src/components/input_fields/image_upload_field.tsx

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) 2018 Ultimaker B.V.
+import * as React from 'react';
+
+import InputFieldWrapper, {InputFieldProps} from './input_field_wrapper';
+import ImageUpload, {ImageFile} from '../image_upload';
+import {Image, ImageShape} from '../image';
+
+
+/**
+ * The image upload field provides these props in addition to those supported by all input fields.
+ */
+export interface ImageUploadFieldProps extends InputFieldProps {
+    /** Size of the image for type image. Include size unit */
+    imageSize: string;
```

Comment:
Before they couldn't be mandatory because the props were applicable to all input types. Now we have a specific type for images we should always pass an image size. Although I understand HTML will default it to the actual size of the image so this can be optional